.travis.yml

```diff
@@ -10,7 +10,10 @@ before_script:
   - cd test/site/www && bower install && cd ../../../
   - phantomjs --webdriver=4444 2>&1 >/dev/null &
 
-script: "sleep 5 && npm run test-snyk && npm run test"
+script:
+    - sleep 5
+    - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then bash npm run test-snyk; fi'
```

Comment:
this needs to be changed to == false, but I want to trigger it and see if things are working like it should at a PR